q2_quality_control/decontam.py

```diff
@@ -104,6 +113,7 @@ def decontam_identify(table: pd.DataFrame,
     metadata = metadata.to_dataframe()
     _check_column_inputs(table, metadata, method, freq_concentration_column,
                          prev_control_column, prev_control_indicator)
+    print("All approriate inputs found")
```

Comment:
We don't need this; the action completing implies this

Reply:
addressed in commit [f31210b](https://github.com/qiime2/q2-quality-control/pull/88/commits/f31210b1b4524e53c8545435910ecb106d40fc8d)